packages/calling-component-bindings/src/callControlSelectors.ts

```diff
@@ -148,3 +149,28 @@ export const devicesButtonSelector: DevicesButtonSelector = reselect.createSelec
     };
   }
 );
+
+/* @conditional-compile-remove(PSTN-calls) */
+/**
+ * Selector type for the {@link HoldButton} component.
+ * @public
+ */
+export type HoldButtonSelector = (
+  state: CallClientState,
+  props: CallingBaseSelectorProps
+) => {
+  checked: boolean;
+  disabled: boolean;
+};
+
+/* @conditional-compile-remove(PSTN-calls) */
+/**
+ * Selector for the {@link HoldButton} component.
+ * @public
+ */
+export const holdButtonSelector: HoldButtonSelector = reselect.createSelector([getCallState], (callState) => {
+  return {
+    checked: callState === 'LocalHold' ? true : false,
+    disabled: callState === 'RemoteHold' ? true : false
```

Comment:
Why can't I hold myself as well if the call is in RemoteHold?
Might be worth adding some documentation here

Reply:
is this something I should omit until we have a answer from calling what the expected behavior is? this isn't something that pertains to PSTN since the local user cannot tell when a PSTN user has them on hold, we never go to this state from what I've seen in my testing.

Reply:
If this doesn't pertain to PSTN, then basis on my interactions with the calling team so far.
Hold is basically a shortcut for disabling audio, video and screenshare stream of a participant.
And should be allowed even if the call is on remote hold. 
I am going to ask calling team for clarification around this anyway.